src/applications/gi/actions/index.js

```diff
@@ -201,7 +201,10 @@ export function fetchProfile(facilityCode, version) {
             .then(zipRatesPayload => {
               withPreview(dispatch, {
                 type: FETCH_PROFILE_SUCCEEDED,
-                payload,
+                payload: {
+                  ...institution,
+                  constants,
```

Comment:
this can be simplified by 
- adding `getState` to the thunk callback on line 180 e.g. `(dispatch, getState) => `
- getting the constants off of the state using a selector that's used in the `mapStateToProps` of the `ProfilePage` 
- adding only the constants the reducer needs- adding the entire constants object to the action obfuscates the specific need e.g.: 
``` 
const { AVGDODBAH, AVGBAH } = state.constants; // whatever the selector is 
// ...
payload: {
  AVGDODBAH,
  AVGBAH,
  ...institution
},
```
- removing the `constants` from the `fetchProfile` parameters

Reply:
Thanks!